ProjectSettings/ProjectSettings.asset

```diff
@@ -774,7 +774,7 @@ PlayerSettings:
   additionalIl2CppArgs: 
   scriptingRuntimeVersion: 1
   gcIncremental: 0
-  assemblyVersionValidation: 1
```

Comment:
Shouldn't need to change this now!

Reply:
Why not? What changed?

(I'll test without this)

Reply:
huh, seems happy now. Not complaining :-)